test/schemaValidation.test.ts

```diff
@@ -1966,5 +1966,17 @@ obj:
       expect(result[0].message).to.eq('Matches multiple schemas when only one must validate.');
       expect(telemetry.messages).to.be.empty;
     });
+
+    it('Unused anchors in an array', async () => {
```

Comment:
This test should be in https://github.com/redhat-developer/yaml-language-server/blob/ba7452ee94105636f7c517394f378d09a71e2f6b/test/yamlValidation.test.ts#L58

Reply:
Sure, I've updated an existing test case to have multiple [un]used anchors handling in an array.